src/components/ActionButton/ActionButton.vue

```diff
@@ -26,38 +26,52 @@ This component is made to be used inside of the [Actions](#Actions) component sl
 
 ```vue
 	<template>
-		<Actions>
-			<ActionButton @click="showMessage('Delete')">
-				<template #icon>
-					<Delete :size="20" />
-				</template>
-				Delete
-			</ActionButton>
-			<ActionButton :close-after-click="true" @click="showMessage('Delete and close menu')">
-				<template #icon>
-					<Delete :size="20" />
-				</template>
-				Delete and close
-			</ActionButton>
-			<ActionButton :disabled="true" @click="showMessage('Disabled')">
-				<template #icon>
-					<Delete :size="20" />
-				</template>
-				Disabled button
-			</ActionButton>
-		</Actions>
+		<div style="display: flex; align-items: center;">
+			<Actions>
+				<ActionButton @click="showMessage('Delete')">
+					<template #icon>
+						<Delete :size="20" />
+					</template>
+					Delete
+				</ActionButton>
+				<ActionButton :close-after-click="true" @click="showMessage('Delete and close menu')">
+					<template #icon>
+						<Delete :size="20" />
+					</template>
+					Delete and close
+				</ActionButton>
+				<ActionButton :close-after-click="true" @click="focusInput">
+					<template #icon>
+						<Plus :size="20" />
+					</template>
+					Create
+				</ActionButton>
+				<ActionButton :disabled="true" @click="showMessage('Disabled')">
+					<template #icon>
+						<Delete :size="20" />
+					</template>
+					Disabled button
+				</ActionButton>
+			</Actions>
+			<input ref="input" />
```

Comment:
@raimund-schluessler why do we have that in the documentation?
I feel this is not relevant with the `ActionButton` at all, no?
Feels like a corner case debug code leftover? :thinking: :see_no_evil:

Reply:
I guess that depends, one could see it as an example how to focus the input field after clicking an action. But it's true, I mainly introduced it to check that focusing actually works.

Reply:
I figured, removed in https://github.com/nextcloud-libraries/nextcloud-vue/pull/4724
I think the example is too broad to have its place here. The nextTick and focus is common usage I would say :)